libs/dal/src/repositories/change/change.schema.ts

```diff
@@ -0,0 +1,41 @@
+import * as mongoose from 'mongoose';
+import { Schema, Document } from 'mongoose';
+import { schemaOptions } from '../schema-default.options';
+import { ChangeEntity } from './change.entity';
+
+const changeSchema = new Schema(
+  {
+    enabled: {
+      type: Schema.Types.Boolean,
+      default: false,
+    },
+    type: Schema.Types.String,
```

Comment:
```suggestion
    type: {
       type: Schema.Types.String
    },
```


From my previous experience using a key named type might cause some issues with mongoose, so wrapping it in object fixes it.